application/values.yaml

```diff
@@ -1,3 +1,6 @@
+activiti-cloud-events-adapter:
+  enabled: true
```

Comment:
@cristina-sirbu @rgauss can we set it by default false (but explicit in the file) I do see a lot of people not using it right now.. as we don't have access to the event gateway just yet.

Reply:
Set enabled to false.